recipe_scrapers/cookwell.py

```diff
@@ -1,7 +1,29 @@
 from ._abstract import AbstractScraper
+import re
 
 
 class CookWell(AbstractScraper):
     @classmethod
     def host(cls):
         return "cookwell.com"
+
+    def nutrients(self):
+        nutrition = None
+        nutrition_pattern = re.compile(
+            r"{\\\"calories\\\":(?P<calories>\d+),\\\"carbohydrates\\\":(?P<carbohydrates>\d+),\\\"fat\\\":(?P<fat>\d+),\\\"protein\\\":(?P<protein>\d+)}"
+        )
+
+        nutrition_script = self.soup.find("script", string=nutrition_pattern)
+        if nutrition_script:
+            nutrition_info = nutrition_pattern.search(
+                nutrition_script.string
+            ).groupdict()
+            if nutrition_info:
+                nutrition = {
```

Comment:
```suggestion
            match = nutrition_pattern.search(nutrition_script.string)
            if match:
                nutrition_info = match.groupdict()
                return {
```

We can skip the assignment to `nutrition_info` and directly check if there is a match here.